sisl/viz/plotly/plotutils.py

```diff
@@ -842,8 +847,7 @@ def trigger_notification(title, message, sound="Submarine"):
         sound_string = f'sound name "{sound}"' if sound else ''
         os.system(f"""osascript -e 'display notification "{message}" with title "{title}" {sound_string}' """)
     else:
-        print(f'Notifications are not implemented in your operating system ({sys.platform}).'
-        ' Maybe consider switching to linux? https://www.amazon.com/s?k=linux+computers&rh=n%3A565108&ref=nb_sb_noss :)')
+        info(f"sisl cannot issue notifications through the operating system ({sys.platform})")
```

Comment:
Hahaha you don't like the unserious messages, I see.

Reply:
Maybe this should be a warning? If they are expecting notifications and we can't deliver them it may be important to say it.

Reply:
But how are these used? I guess notifications do not endanger any usage or functionality, just a _heads up_?

Reply:
https://github.com/zerothi/sisl/pull/302#discussion_r574643277 And no unserious messages and weird images don't below in these projects ;)

While it may be fun, you never know who is at the receiving end, so there is no need to take the risk of insulting users.

Reply:
This is not at the core of anything right now. It's just a possibility when you are listening to changes on the files that a plot uses. When the plot is updated, you can get notified. But this might not even be necessary (I'm sure it's not), I thought it was just a fun feature to add (that's why it is kind of funky, calling shell commands directly).

Regarding info/warning yeah maybe it's not that important, it would be a bit weird to not be receiving notifications if you specified that you wanted to receive them but that's ok.